packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -16,13 +16,16 @@ import {LinkFormatButton, RemoveLinkFormatButton} from './toolbar/linkButton'
 import {TableMenu} from './toolbar/tableMenu'
 import {WepublishEditor} from './editor/wepublishEditor'
 
-export type RichTextBlockProps = BlockProps<RichTextBlockValue>
+export interface RichTextBlockProps extends BlockProps<RichTextBlockValue> {
+  displayOnly: boolean
+}
 
 export const RichTextBlock = memo(function RichTextBlock({
   value,
   autofocus,
   disabled,
-  onChange
+  onChange,
+  displayOnly
```

Comment:
add a default value `displayOnly = false`

Reply:
Do we need this additional arg? Couldn't just say if `disabled` then hide toolbar?

Reply:
@roosnic1 why add default value false?

Reply:
@bishoymelek because `displayOnly` is a boolean and shouldn't be `undefined`